netbox/virtualization/tables/virtualmachines.py

```diff
@@ -84,20 +86,26 @@ class VirtualMachineTable(TenancyColumnsMixin, ContactsColumnMixin, NetBoxTable)
     interface_count = tables.Column(
         verbose_name=_('Interfaces')
     )
+    virtual_disk_count = tables.Column(
+        verbose_name=_('Virtual Disks')
+    )
     config_template = tables.Column(
         verbose_name=_('Config Template'),
         linkify=True
     )
+    disk_size = tables.Column(
```

Comment:
Similar to the API serializer, it might be cleaner to override the `disk` column to return the concrete field if set, and otherwise fall back to reporting the aggregate size of all attached VirtualDisks. This can be normalized once we've removed the `disk` field in a future release.